src/utilities.ts

```diff
@@ -73,7 +74,7 @@ export function isDockerInstalled(outputChannel: vscode.OutputChannel, cb: Funct
 }
 
 export function isAnsibleInstalled(outputChannel: vscode.OutputChannel, cb: Function): void {
-    var cmd = process.platform === 'win32' ? 'ansible --version' : 'type ansible';
+    var cmd = process.platform === 'win32' ? 'wsl -- ansible --version' : 'type ansible';
```

Comment:
if wsl is  not enabled?

Reply:
wsl is for windows 10 and later. there's option to run ansible on cygwin on windows.

Reply:
While I agree that cygwin does make it possible to run Ansible on Windows, instructions for installing Ansible on WSL is actually documented on the Ansible docs site. https://docs.ansible.com/ansible/latest/user_guide/windows_faq.html#can-ansible-run-on-windows

If this is really wanted & used by the community, I'll gladly put that check back in.